h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -1374,10 +1381,11 @@ private void freeUnusedChunks() {
         return collector.getReferenced();
     }
 
-
+    final ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
`executorService.shotdown()` should be invoked somewhere.

If `submit(...)` was used, resources (threads) are not released without proper shutdown.